data/human/ships.txt

```diff
@@ -2608,6 +2608,64 @@ ship "Nest"
 
 
 
+ship "Nighthawk"
```

Comment:
I think we should define a variant thats max crewed and only use that one since this is a GA ship (just write `crew 28` on the first line of said variant)

Reply:
?